### ISSUE TITLE
NVDA doesn't read the WindowsLiveMail arabic message content accurately.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3131]
**Reported by fatma.mehanna on 2013-04-05 21:02**
hi,
this is a feedBack from arabic users 
when they want to read the message content, the mesage they are composing not any message, line by line, NVDA reads 2 or 3 words of the line and stops reading unless they put the cursor at the end of line.
this problem doesn't occure when they use the sayAll command. the whole content is being read correctly.
this issue is applied to braille and speach.

i will attach a log file in a .txt file as they paste the log in their email message.
thanks.

**Comment by mohdshara** [comment]
This should be closed as won't fix like all other Windows Live issues.

**Comment by leonardder** [comment]
Thanks for pointing this out.
